src/libraries/System.Private.Xml/tests/XPath/XPathDocument/System.Xml.XPath.Tests.csproj

```diff
@@ -1,196 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
note there were previously 3 csproj files testing different implementations of XPath with some subtle differences, now we only have two kinds. I cannot remember exactly what the differences were but I do remember those tests actually found a bunch of different issues and removing one kind might reduce coverage significantly (I might remember incorrectly, it's been couple of years since I touched this)

Reply:
what 3 csproj says you referring to?

Reply:
Bad memory I guess, just validated and there are in fact 2